packages/ses/src/global-object.js

```diff
@@ -95,3 +91,25 @@ export const setGlobalObjectMutableProperties = (
     }
   }
 };
+
+/**
+ * setGlobalObjectEvaluators()
+ * Set the eval and the Function evaluator on the global object with given evalTaming policy.
+ *
+ * @param {Object} globalObject
+ * @param {Function} evaluator
+ */
+export const setGlobalObjectEvaluators = (globalObject, evaluator) => {
```

Comment:
This dropped the `markVirtualizedNativeFunction` for `eval` and `Function` values.